osu.Game/Overlays/Options/OptionsSubsection.cs

```diff
@@ -28,7 +29,7 @@ public OptionsSubsection()
                     Margin = new MarginPadding { Bottom = 10 },
                     Font = @"Exo2.0-Black",
                 },
-                Content = new FillFlowContainer

```

Comment:
It does.
Still, the original code crashes. It's weird because setting a getter only proprierty is done elsewhere without crashing.